config/overrides.go

```diff
@@ -113,6 +113,8 @@ var groupMap = map[string]GroupKindCalculator{
 	"google_document_ai.+": ReplaceGroupWords("", 2),
 
 	"google_essential_contacts_contact$": ReplaceGroupWords("", 2),
+
+	"google_cloud_quotas.+$": ReplaceGroupWords("", 2),
```

Comment:
This little bit is underdocumented in the https://github.com/crossplane/upjet/blob/main/docs/adding-new-resource.md#gcp guide fwiw, but adding this induced the code generation for a new provider as well.